api-test.json

```diff
@@ -14905,202 +15475,1481 @@
-																	"    pm.response.to.have.header(\"Location\");\r",
-																	"    pm.expect(pm.response.headers.get(\"Location\")).to.include(subId);\r",
-																	"});\r",
-																	"setTimeout(function(){\r",
-																	"    \r",
-																	"    pm.sendRequest(notificationServer + \"/\" + subId, function (err, response) {\r",
-																	"            pm.test(\"Check for notification\", function () {\r",
-																	"                var value = response.json();\r",
-																	"                console.log(\"Retrieving notifications from testserver\")\r",
-																	"                console.log(notificationServer + \"/\" + subId)\r",
-																	"                console.log(value);\r",
-																	"                pm.expect(value.length).to.greaterThan(2);\r",
-																	"                for(let i=0; i < 3; i++){\r",
-																	"                    notification = value[i].body;\r",
-																	"                    headers = value[i].headers;\r",
-																	"                    pm.expect(notification.type).to.eql(\"Notification\");\r",
-																	"                    pm.expect(notification).to.have.property(\"id\");\r",
-																	"                    pm.expect(notification.data.length).to.eql(1);\r",
-																	"                    var notificationData = notification.data[0];\r",
-																	"                    delete notificationData[\"createdAt\"];\r",
-																	"                    delete notificationData[\"modifiedAt\"];\r",
-																	"                    console.log(\"expected notification: \" + JSON.stringify(payloadBody));\r",
-																	"                    console.log(\"received notification: \" + JSON.stringify(notificationData));\r",
-																	"                    pm.expect(notificationData).to.eql(created);\r",
-																	"                }\r",
-																	"            });\r",
-																	"        });\r",
-																	"    }, 15000);\r",
-																	""
+															"name": "Create Subscription With Id No Context",
+															"event": [
+																{
+																	"listen": "test",
+																	"script": {
+																		"exec": [
+																			"var payloadBody = JSON.parse(pm.request.body.raw);\r",
+																			"var subId = payloadBody.id;\r",
+																			"pm.collectionVariables.set(\"subId\", subId);\r",
+																			"pm.test(\"Status code is 201\", function () {\r",
+																			"    pm.response.to.have.status(201);\r",
+																			"});\r",
+																			"\r",
+																			"pm.test(\"Check location header\", function () {\r",
+																			"    pm.response.to.have.header(\"Location\");\r",
+																			"    pm.expect(pm.response.headers.get(\"Location\")).to.include(subId);\r",
+																			"});"
+																		],
```

Comment:
Add test case for validate response data.

Reply:
done